tests/org.eclipse.jface.tests/src/org/eclipse/jface/tests/viewers/TreeViewerTest.java

```diff
@@ -50,4 +56,25 @@ protected String getItemText(int at) {
 		return tree.getItems()[at].getText();
 	}
 
+	@Test
+	public void testAutoExpandTrivialPathThroughEvent() {
```

Comment:
```suggestion
	public void testAutoExpandOnSingleChildThroughEvent() {
```

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/fb256ed9423ce9f552aa278fa3893d833ccc0e63